platform/linux-generic/arch/aarch64/odp/api/abi/atomic_inlines.h

```diff
@@ -152,7 +152,64 @@ static inline int _odp_atomic_cas_acq_rel_u128(odp_atomic_u128_t *atom, odp_u128
 
 	return 0;
 }
-#else /* _ODP_LOCK_FREE_128BIT_ATOMICS */
+
+static inline void _odp_atomic_add_u32(odp_atomic_u32_t *atom, uint32_t val)
```

Comment:
The commit subject line could be tweaked a bit:
`linux-gen: atomic: use STADD explicitly for aarch64 atomic add operations`

Reply:
Sure, I will make the change. Thanks Matias!